### DISCUSSION TITLE
Zooz ZEN73 / ZEN74 support

### DISCUSSION THREAD

**Discussion by jspanitz** [zwave-js/zwave-js#2519] [Q&A]
Just added a few ZEN74 switches to my install and found they are not supported.  I'd love to request they be added.  I also realize this is community supported so I'd like to help, just need a guide on how to do so or someone to tell me what information to provide.

**Comment by AlCalzone** [comment]
Hey,

if you want to provide a config file yourself, it's probably best if you start by reading through the documentation on config files, which starts here:
https://zwave-js.github.io/node-zwave-js/#/config-files/overview

If that's too much to ask, we have an issue template for missing config files, which tells you which info we (or someone who wants to help out) need to add it.
